tests/unit/test_data_transformer.py

```diff
@@ -0,0 +1,384 @@
+        """Test 'inverse_transform' on a np.ndarray representing one continuous and one
+        discrete columns.
+
+        It should use the appropriate '_fit' type for each column and should return
+        the corresponding columns. Since we are using the same example as the 'test_transform',
+        and these two functions are inverse of each other, the returned value here should
+        match the input of that function.
+
+        Setup:
+            - Mock _column_transform_info_list
+            - Mock _inverse_transform_discrete
+            - Mock _inverse_trarnsform_continuous
+
+        Input:
+            - column_data = a concatenation of two np.ndarrays
+              - the first one refers to the continuous values
+                - the first column contains the normalized values
+                - the remaining columns correspond to the a one-hot
+              - the second one refers to the discrete values
+                - the columns correspond to a one-hot
+        Output:
+            - numpy array containing a discrete column and a continuous column
+
+        Side Effects:
+            - _transform_discrete and _transform_continuous should each be called once.
+        """
+
+    def test_convert_column_name_value_to_id(self):
+        """Test 'convert_column_name_value_to_id' on a simple '_column_transform_info_list'.
+
+        Tests that the appropriate indexes are returned when a table of three columns,
+        discrete, continuous, discrete, is passed as '_column_transform_info_list'.
+
+        Setup:
+            - Mock _column_transform_info_list
+
+        Input:
+            - column_name = the name of a discrete column
+            - value = the categorical value
+
+        Output:
+            - dictionary containing:
+              - 'discrete_column_id' = the index of the target column,
+                when considering only discrete columns
+              - 'column_id' = the index of the target column
+                (e.g. 3 = the third column in the data)
+              - 'value_id' = the index of the indicator value in the one-hot encoding
+        """
+        ohe = Mock()
+        ohe.transform.return_value = np.array([
+            [0, 1]  # one hot encoding, second dimension
+        ])
+        transformer = DataTransformer()
+        transformer._column_transform_info_list = [
+            ColumnTransformInfo(
+                column_name='x', column_type='continuous', transform=None,
+                transform_aux=None,
+                output_info=[SpanInfo(1, 'tanh'), SpanInfo(3, 'softmax')],
+                output_dimensions=1 + 3
+            ),
+            ColumnTransformInfo(
+                column_name='y', column_type='discrete', transform=ohe,
+                transform_aux=None,
+                output_info=[SpanInfo(2, 'softmax')],
+                output_dimensions=2
+            )
+        ]
+        result = transformer.convert_column_name_value_to_id('y', 'yes')
```

Comment:
add a white space to keep the same style where we split the asserts and run results. (as below)